modules/salt/pages/yomi.adoc

```diff
@@ -0,0 +1,174 @@
+[[yomi.installer]]
+= Install with Yomi
+
+Yomi (yet one more installer) is an installer for openSUSE operating systems.
+It is a SaltStack state, and can be used for installing Salt clients.
+
+Yomi consists of two components:
+
+* The Yomi formula, which contains the Salt states and modules required to drive the installation.
+For the formula, see https://build.opensuse.org/package/show/systemsmanagement:yomi/yomi-formula.
```

Comment:
This URL is correct. But we managed to add the formula as a package for Uyuni. The user can install it now via `zypper install yomi-formula`